src/v2/view-builder/views/ov/ChallengeOktaVerifyFastPassView.js

```diff
@@ -130,36 +125,14 @@ const Body = BaseFormWithPolling.extend(Object.assign(
     },
 
     showCustomFormErrorCallout(error) {
-      const errorSummaryKeys = error?.responseJSON?.errorSummaryKeys;
-      const isBiometricsRequiredMobile = errorSummaryKeys 
-          && errorSummaryKeys.includes(OV_UV_ENABLE_BIOMETRICS_FASTPASS_MOBILE);
-      const isBiometricsRequiredDesktop = errorSummaryKeys 
-          && errorSummaryKeys.includes(OV_UV_ENABLE_BIOMETRICS_FASTPASS_DESKTOP);
-
-      if (isBiometricsRequiredMobile || isBiometricsRequiredDesktop) {
-        const bulletPoints = [
-          loc('oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.point1', 'login'),
-          loc('oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.point2', 'login'),
-          loc('oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.point3', 'login')
-        ];
-
-        // Add an additional bullet point for desktop devices
-        if (isBiometricsRequiredDesktop) {
-          bulletPoints.push(
-            loc('oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.point4', 'login')
-          );
-        }
-
-        const options = {
-          type: 'error',
-          className: 'okta-verify-uv-callout-content',
-          title: loc('oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.title', 'login'),
-          subtitle: loc('oie.authenticator.oktaverify.method.fastpass.verify.enable.biometrics.description', 'login'),
-          bullets: bulletPoints,
-        };
-        this.showMessages(createCallout(options));
-        return true;
+      const options = getBiometricsErrorOptions(error, false);
+      
+      if (_.isEmpty(options)) {
```

Comment:
in which case it will be empty ?

Reply:
it shouldn't be empty as of current usages. This is more like a safety check for future usage.

Reply:
you can remove this..

Reply:
Correction: if the error is not biometrics error, options will be empty so we need to keep this change.